lib/logstash/outputs/influxdb.rb

```diff
@@ -126,21 +126,26 @@ def receive(event)
     # ]
     event_hash = {}
     event_hash['name'] = event.sprintf(@series)
+
     sprintf_points = Hash[@data_points.map {|k,v| [event.sprintf(k), event.sprintf(v)]}]
     if sprintf_points.has_key?('time')
-      @logger.error("Cannot override value of time without 'allow_override_time'. Using event timestamp") unless @allow_override_time
+      unless @allow_time_override
+        logger.error("Cannot override value of time without 'allow_time_override'. Using event timestamp")
+        sprintf_points['time'] = event.timestamp.to_i
+      end
     else
       sprintf_points['time'] = event.timestamp.to_i
     end
```

Comment:
no because if `sprintf_points` has key "time" and `@allow_time_override` is `true` it will reset to the   `event.timestamp.to_i` which is not what you want - you want to do nothing and let  `sprintf_points["time"]` as it is.